calyx/src/passes/remove_if.rs

```diff
@@ -88,12 +88,18 @@ impl Visitor for RemoveIf {
                     }
                 }
 
-                let mut comps = vec![];
-                comps.append(&mut con.cond);
-                comps.append(&mut tbranch.comps.clone());
-                comps.append(&mut fbranch.comps.clone());
 
-                Ok(Action::Change(ast::Control::enable(comps)))
+                let mut comps_seq = vec![];
+                let mut comps1 = vec![];
+                let mut comps2 = vec![];
+                comps1.append(&mut con.cond);
+                comps2.append(&mut tbranch.comps.clone());
+                comps2.append(&mut fbranch.comps.clone());
+                comps_seq.push(ast::Control::enable(comps1) );
```

Comment:
Is there a reason to create a new vector `comps1`, append all the elements in `con.cond` into and then pass it to `comps_seq.push(enable(comps1)`.

Can't you do `comps_seq.push(enable(con.cond.clone()))`?

Reply:
updated